Pipfile.lock

```diff
@@ -57,22 +57,6 @@
             "index": "pypi",
             "version": "==0.14"
         },
-        "funcsigs": {
-            "hashes": [
-                "sha256:330cc27ccbf7f1e992e69fef78261dc7c6569012cf397db8d3de0234e6c937ca",
-                "sha256:a7bb0f2cf3a3fd1ab2732cb49eba4252c2af4240442415b4abce3b87022a8f50"
-            ],
-            "markers": "python_version < '3.3'",
-            "version": "==1.0.2"
-        },
-        "futures": {
```

Comment:
This is a backport of https://docs.python.org/3/library/concurrent.futures.html for 2.7 so unneeded